biochemist-python/chapters/introduction.ipynb

```diff
@@ -85,23 +85,12 @@
   },
```

Comment:
<p>Again, not a fan of the variable name <code>v_atorbelowkm</code>.  Python style guides suggest separating words with underscores for clarity. I think removing the underscores only very slightly reduces typing and greatly reduces clarity. Maybe the name could be less descriptive like <code>v_below_km</code>  , <code>below_km</code>, <code> v_filtered</code> , etc</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MolSSI-Education/python-scripting-biochemistry/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>